src/tabs/tabs.js

```diff
@@ -72,9 +63,35 @@ angular.module('ui.bootstrap.tabs', [])
     ctrl.tabs.splice(index, 1);
   };
 
+  ctrl.updateActiveIndex = function updateActiveIndex(tab) {
+    if (tab.active) {
+      ctrl.active = tab.index;
+      oldIndex = tab.index;
+      ctrl.tabs.sort(function(t1, t2) {
```

Comment:
In some cases, the tabs would get out of order. Although, the tabs don't necessarily need to get sorted at all in the first place but the tests assume an ordering of tabs